test/e2e/e2eutil/wait_util.go

```diff
@@ -240,3 +240,23 @@ func waitPodsDeleted(kubecli kubernetes.Interface, namespace string, timeout tim
 	})
 	return pods, err
 }
+
+func WaitUntilPodsReady(kubecli kubernetes.Interface, size int, namespace string, timeout time.Duration, lo metav1.ListOptions) error {
```

Comment:
`size` is not needed, right?
Also switch `timeout` and `lo` position? Group all k8s stuff together is better.